### ISSUE TITLE
[Suggestion] Fast auto crafting for enhanced farming

### ISSUE THREAD

**Issue by nikisalli** [MeteorDevelopment/meteor-client#1091]
A tool to auto craft a recipe when possible, for example auto craft bones into bonemeal when you got bones in the inventory and there is enough inventory space to hold the resulting items.

**Comment by NoobMaster420-69** [comment]
Yes i want this would be super useful for crafting axes really fast in test 2b and mabye add support to craft the stuff needed to make the thing you want to craft

**Comment by Tokara1** [comment]
Try out itemscroller mod by marouhon (https://github.com/maruohon/itemscroller)

You can fast craft With It but be careful With Packets bc otherwise youll fly to disconect

**Comment by dadudeme** [comment]
> be careful With Packets bc otherwise youll fly to disconect

there is an option to back up packets to stop that

**Comment by RedCarlos26** [comment]
Duplicate of #1038
